src/main/java/nextstep/ladder/domain/Lines.java

```diff
@@ -0,0 +1,66 @@
+package nextstep.ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Lines {
+
+    private List<Line> lines;
+
+    public Lines(int maxNum, LineStrategy lineStrategy) {
+        this.lines = new ArrayList<>();
+        this.lines = drawLine(maxNum, lineStrategy);
+    }
```

Comment:
```suggestion
    private Lines(List<Line> lines) {
        this.lines = lines;
    }
    
    public static Lines of (List<Line> lines) {
        return new Lines(lines);
    }
    
    public static Lines of (int maxNum, LineStrategy lineStrategy) {
        return of(drawLine(maxNum, lineStrategy));
    }
```
이렇게 정적 생성 팩토리 메소드 패턴을 이용해주면 더 깔끔할 것 같아요!

Reply:
> 이 처리를 안해주면 nullPointerException 에러를 뱉어서 객체를 먼저 만들어주었습니다

Reply:
https://github.com/next-step/java-ladder/pull/985#discussion_r684221221

Reply:
전부 다 static 메소드로 바꾸고 필드 변수마저 static으로 바꿔야할텐데 괜찮은 부분일까요?